quinn-proto/src/endpoint.rs

```diff
@@ -29,7 +29,7 @@ use crate::{
         ConnectionEvent, ConnectionEventInner, ConnectionId, DatagramConnectionEvent, EcnCodepoint,
         EndpointEvent, EndpointEventInner, IssuedCid,
     },
-    token::TokenDecodeError,
+    token::ValidationError,
```

Comment:
Either name this `TokenValidationError`, or always refer to it qualified as `token::ValidationError`. Bare `ValidationError` is ambiguous outside of its module of origin.

Reply:
Leaving as `ValidationError`, and changed the (1) usage in `endpoint.rs` to specify `token::ValidationError::Ignore`. Because:

- `TokenValidationError` is long.
- I anticipate future PRs likely being able to make this error entirely private to `token.rs` anyways.

Reply:
I've always liked terse names that get their context from the module system anyway.